charts/misc-apps/values.yaml

```diff
@@ -135,7 +135,7 @@ argocdApps:
   # -- Chart
   chart: argocd-apps
   # -- [argocd-apps Helm chart](https://github.com/argoproj/argo-helm/tree/main/charts/argocd-apps) version
-  targetRevision: 2.0.0
+  targetRevision: 2.0.2
```

Comment:
Either this should be 2.0.1 or the PR description should be updated to 2.0.2

Reply:
I updated the title after rechecking the upstream chart.